public/manifest-assets/site.webmanifest

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "NEAR.REST",
```

Comment:
Shouldn't this be NEARREST NEIGHBOR?

Reply:
This when added to home page. I can easily change, but thought NEAR.REST would be more appropriate. Also NEARREST NEIGHBOR looked a bit too long on my homescreen compared to everything else.